### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by EmmaTellblom**

### CODE REVIEW

**Comment by EmmaTellblom** [comment]
@carstenhag What do you think? I was a bit unsure on what to have on the 'additional' template...

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+**Expected behavior**
+A clear and concise description of what was expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Smartphone information (please complete the following information):**
+ - Device: [e.g. S24]
+ - Android version: [14]
+ - App-Version [22]
```
Comment (carstenhag) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]:
```suggestion
 - App version: [22]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Describe how we can reproduce the error ourselves.
+
+**Expected behavior**
+A clear and concise description of what was expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Smartphone information (please complete the following information):**
+ - Device: [e.g. S24]
```
Comment (carstenhag) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]:
```suggestion
 - Device: [e.g. Samsung Galaxy S24]
```

**Comment by carstenhag** [comment]
Looks good to me, not sure if we need newlines after the `**Headlines**` but we will just see I guess

**Review by carstenhag — APPROVED** [review]

**Pull request merged**